src/lib/components/consignments/__stories__/consignments-bottom-aligned.story.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from "react"
+import { Text, View } from "react-native"
+
+import Search from "../components/artist-search-results"
+import BottomAlignedButton, { BottomAlignedProps } from "../components/bottom-aligned-button"
+
+export const name = "Consignments - bottom aligned"
+export const component = BottomAlignedButton
+
+interface States {
+  [name: string]: BottomAlignedProps
+}
```

Comment:
This hooks up the prop states that we're going to present next to the props. What this interface says is: this object is something that will have string (keys) that will always be of the (value) type `BottomAlignedProps`

```ts
const thing = { "This thing": [an object conforming to BottomAlignedProps] }
```

is a legit `States`. Why do I do this? Well further down you'll see that I used to have to declare then prop types everywhere, but with this, I can say the `allStates` array below is an array of these and now (through inference) all objects added to that const will be checked against `BottomAlignedProps`. If someone adds a new prop that _has_ to be included, they'll get compiler errors from TypeScript on the old states, ensuring the code is always 💯.